Tools/dockerfiles/Dockerfile

```diff
@@ -1,20 +1,20 @@
 FROM mono:4.2
 
-# all in one RUN to produce less, smaller layers
+ARG AUTOREST_VERSION=0.16.0
+
+ADD autorest /opt/autorest/autorest
+
+# all-in-one RUN to produce smaller image with less layers
 RUN apt-get -qqy update && \
 	apt-get -qqy upgrade && \
-	apt-get -qqy install wget unzip && \
+	apt-get -qqy install bsdtar && \
```

Comment:
No need for wget when we have curl, and bsdtar is much smaller + supports piping the data from curl.